docs/windows.md

```diff
@@ -1,142 +1,139 @@
 # Building on Windows
-
-## Install Git For Windows
-
-As Git is used by the Go language to download dependencies, you need
-to install Git, for example from http://git-scm.com/.
-
-You also must make sure that the directory that contains the Git For
-Windows binary is in the Path environment variable. Note that Git For
-Windows has a 'git' binary in a 'Git\bin' directory and another one in
-a 'Git\cmd' directory. You should only put the 'Git\cmd' directory in
-the Path environment variable.
+![](https://ipfs.io/ipfs/QmccXW7JSZMVXidSc7tHsU6aktuaiV923q4yBGHUsdymYo/build.gif)
 
 ## Install Go
+`go-ipfs` is built on Golang and thus depends on it for all building methods.  
+https://golang.org/doc/install  
+The `GOPATH` environment variable must be set as well.  
+https://golang.org/doc/code.html#GOPATH
 
-Please install the Go language as explained on
-https://golang.org/doc/install.
-
-To properly install Go, you will need to set some environment
-variables. We recommend you to set them globally using the Control
-Panel, as explained in the documentation above, so that these
-environment variables are automatically available in all the possible
-environments that you might want to use like Git Bash, Windows's cmd,
-Cygwin's terminal, Windows' PowerShell and so on.
-
-You must make sure that the GOROOT environment variable is set and
-that the %GOROOT%/bin directory is in the Path environment variable.
-
-The GOPATH environment variable should also be set to a directory that
-you have created, and the %GOPATH/bin directory should also be in the
-Path environment variable.
+## Choose the way you want to proceed
+`go-ipfs` utilizes `make` to automate builds and testing, but can be built without it using only `git` and `go`.  
+No matter which section you choose, if you encounter issues, please see the [Troubleshooting](#troubleshooting) section.  
 
-## Download go-ipfs and fix Git authentication
+Using `make`:  
+These sections cover setting up the build environment, each option is equally valid and differ only in their setup process. If you have no preference or special requirements, MSYS2 is recommended as the first choice.  
```

Comment:
Hopefully it’s already clear to people that this is about setting up their build environment :)  The thing that might be more useful to clarify here is what MSYS2 and Cygwin are, e.g:

> MSYS2 and Cygwin make it possible to run Unix-style programs (like `make`) on Windows. Both are great, but if you aren’t already using one, we recommend MSYS2.

Reply:
Sometimes I'm a little redundant and robotic...
I like the more human friendly wording here but I have qualms with saying Cygwin is "great" ;^)
How does this revision sound?
>MSYS2 and Cygwin provide the Unix tools we need to build `go-ipfs`. You may use either, but if you don't already have one installed, MSYS2 is recommended.

P.S.
I'm a little robotic and redundant sometimes.